google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java

```diff
@@ -854,7 +865,9 @@ public HttpResponse execute() throws IOException {
     Preconditions.checkNotNull(requestMethod);
     Preconditions.checkNotNull(url);
 
+    Span span = tracer.spanBuilder(traceSpanNamePrefix + "execute").startSpan();
```

Comment:
traceSpanNamePrefix + "execute" this is a constant string, you should use a constant for this to avoid unnecessary overhead.